airflow/utils/types.py

```diff
@@ -22,3 +22,13 @@ class DagRunType(enum.Enum):
     BACKFILL_JOB = "backfill"
     SCHEDULED = "scheduled"
     MANUAL = "manual"
+
+    @staticmethod
+    def resolve_run_type(run_id: str) -> "DagRunType":
```

Comment:
```suggestion
    def from_run_id(run_id: str) -> "DagRunType":
```

i.e. `DagRunType.from_run_id(dag_run.run_id)`

Reply:
Agree, done